### PR TITLE
Initial display of games using test data

### PR DESCRIPTION
**Pull request by madagas-karrh**
# Summary of changes
Added arclightning/games.html which displays a grid of the game thumbnails in the sample games array

# How was this tested?
Checked on multiple browsers at multiple sizes, even though our machine has a constant size

# TODO:
Currently does not display title, the div is simply filled by the image. Need to figure out how each div should be designed.

#10

### CODE REVIEW

File: arclightning_frontend/games.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+for(var key in gameObjects){
+	gameObjectsArray.push(gameObjects[key]);
+}
+
+
+//TODO: instead of creating a grid of divs maybe I will create vue components for each of the game cells
+function loadGames(){
+	var rowNum = 5; //will be number of games/3 
+	var colNum = 3;  //set to 3, could be any number of columns
+	for(var i = 0; i < rowNum; i++){
```
Comment (starfys) [inline arclightning_frontend/games.js]: This works, though we should probably use Vue's system for this

**Comment by starfys** [comment]
https://vuejs.org/v2/guide/list.html

I think this API is what we should use for this. In addition, I read that bootstrap will wrap columns to the next row if they go over. So you could use this to create a bootstrap column for every item, and bootstrap will handle the wrapping for you.

**Comment by madagas-karrh** [comment]
games.html now contains a script tag where vuejs is used to display the thumbnails of each of the games in the array and external javascript file (games.js) has been removed.
Aesthetically, I still do not have the game cells displaying any of the other game information. Could you give me an example of what you want each cell to look like, please?

**Comment by starfys** [comment]
Will do. I'd post an image here, but github is weird, so I'll post it to the Slack.

**Comment by madagas-karrh** [comment]
I fixed the formatting of the display of games.html

**Comment by madagas-karrh** [comment]
Fixed the font issue!

**Comment by starfys** [comment]
awoooo

**Pull request merged**